src/pages/index.js

```diff
@@ -229,5 +229,7 @@ function handleSubmit(request, modalInstance) {
   request()
     .then(() => modalInstance.close())
     .catch(console.error)
-    .finally(() => modalInstance.renderLoading(false));
+    .finally(() => {
+        formValidators[modalInstance.formName].resetValidation();
```

Comment:
`  formValidators[modalInstance.formName].resetValidation();` should be in `then` to disable the button only after a successful response to be able to click the button in case of a server error